internal/photoprism/index_mediafile.go

```diff
@@ -352,6 +352,9 @@ func (ind *Index) MediaFile(m *MediaFile, o IndexOptions, originalName string) (
 			details.SetSubject(metaData.Subject, entity.SrcXmp)
 			details.SetArtist(metaData.Artist, entity.SrcXmp)
 			details.SetCopyright(metaData.Copyright, entity.SrcXmp)
+
+			// Update externally marked as favorite.
+			photo.SetFavorite(metaData.Favorite)
```

Comment:
Won't this also automatically UNSELECT manually selected favorites?

Reply:
You mean in the case when a photo has been made a favorite through PhotoPrism and is then re-indexed but without a side-car that marks it as favorite? It would be a similar problem for the reverse case, if a side-car file exists denoting it as favorite but it has been purposefully unselected from the UI.

Would a reasonably first step be to accept the latter for now by doing this?
```suggestion
			if metaData.Favorite {
				photo.SetFavorite(metaData.Favorite)
			}	
```